pennylane_lightning/lightning_gpu/lightning_gpu.py

```diff
@@ -679,9 +808,9 @@ def expval(self, observable, shot_range=None, bin_size=None):
                 or (observable.arithmetic_depth > 0)
                 or isinstance(observable.name, List)
             ):
-                ob_serialized = QuantumScriptSerializer(self.short_name, self.use_csingle)._ob(
-                    observable, self.wire_map
-                )
+                ob_serialized = QuantumScriptSerializer(
+                    self.short_name, self.use_csingle, self._mpi
+                )._ob(observable, self.wire_map)
```

Comment:
Why are we accessing the private method in here? Isn't there a public-facing method we can use?

Reply:
Good question. It is inherited from `lightning.kokkos` and `lightning.qubit` [here](https://github.com/PennyLaneAI/pennylane-lightning/blob/cd0cfc078b64e245b2464ec4ced635e19ed4a75c/pennylane_lightning/lightning_qubit/lightning_qubit.py#L452). Not sure why. Could @vincentmr or @AmintorDusko explain it to us?

Reply:
Same, I took that from @AmintorDusko 's work.

Reply:
Let's track this to update the interface later, as we should avoid accessing privates where possible.

Reply:
I think it would be a good call to have a public method here.
Feel free to update that.